public/app/features/alerting/unified/components/rule-viewer/RuleViewer.test.tsx

```diff
@@ -1,16 +1,26 @@
+import 'whatwg-fetch';
 import { render, waitFor, screen } from '@testing-library/react';
```

Comment:
This can be tidied up a little bit now after https://github.com/grafana/grafana/pull/85672, I can add the changes into here if you don't mind, it's a small tweak